manifest.json

```diff
@@ -243,6 +243,19 @@
             }
           }]
         },
+        {
+          "platforms": [{
+            "dockerfile": "src/fedora/rawhide/amd64",
+            "os": "linux",
+            "osVersion": "fedora-rawhide",
+            "tags": {
+              "fedora-rawhide-$(System:TimeStamp)-$(System:DockerfileGitCommitSha)": {},
+              "fedora-rawhide": {
```

Comment:
This isn't necessary.  It would only be needed if there was another Dockerfile in the build that needed to use this as a base image.